nirc_ehr/resources/data/acquistion_type.tsv

```diff
@@ -0,0 +1,6 @@
+value	title
```

Comment:
Title of this file is misspelled. Also needs to be added to lookupsManifest, lookupsManifestTest and editable_lookups.

Reply:
Good catch, now fixed!
since i cherry picked changes from a different fb into this fb, looks like i missed to get the lookup config files.